### DISCUSSION TITLE
DDL Scripts for Creation

### DISCUSSION THREAD

**Discussion by Chandrabantu** [TPC-Council/HammerDB#655] [Q&A] [answered]
To test the Performance of Oracle DB, my DBA is not willing to share System User and Password or Create a new user with Same Previleage. Is there a way where I can send scripts to DBA to create schemas, tables, stored procedures etc and then we can run Test driver script, load script virtual users on GUI

Please help

**Comment by sm-shaw** [comment, marked as answer]
Your best approach is to ask the DBA to run the schema build for you using the GUI https://www.hammerdb.com/docs/ch04s03.html and then share the created user and password for the schema they have created. You can then run the workloads using this user and password.

**Reply by Chandrabantu** [reply]
Thanks Steve for Prompt Response
